gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -17,41 +17,49 @@
 import gtsam
 import numpy as np
 from gtsam import (
-    MFAS,
-    BinaryMeasurementPoint3,
     BinaryMeasurementsPoint3,
+    BinaryMeasurementPoint3,
     BinaryMeasurementsUnit3,
     BinaryMeasurementUnit3,
+    MFAS,
     Point3,
     Pose3,
     Rot3,
+    symbol_shorthand,
     TranslationRecovery,
     Unit3,
 )
-from scipy import stats
 
-import gtsfm.utils.coordinate_conversions as conversion_utils
+import gtsfm.common.types as gtsfm_types
 import gtsfm.utils.geometry_comparisons as comp_utils
 import gtsfm.utils.logger as logger_utils
 import gtsfm.utils.metrics as metrics_utils
+import gtsfm.utils.sampling as sampling_utils
 from gtsfm.averaging.translation.translation_averaging_base import TranslationAveragingBase
 from gtsfm.common.pose_prior import PosePrior
+from gtsfm.common.sfm_track import SfmTrack2d
 from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
 
 # Hyperparameters for 1D-SFM
 # maximum number of times 1dsfm will project the Unit3's to a 1d subspace for outlier rejection
 MAX_PROJECTION_DIRECTIONS = 2000
 OUTLIER_WEIGHT_THRESHOLD = 0.125
-MAX_KDE_SAMPLES = 2000
 
 NOISE_MODEL_DIMENSION = 3  # chordal distances on Unit3
 NOISE_MODEL_SIGMA = 0.01
 HUBER_LOSS_K = 1.345  # default value from GTSAM
 
 MAX_INLIER_MEASUREMENT_ERROR_DEG = 5.0
 
+TRACKS_TO_CAMERAS_RATIO = 20
+
 logger = logger_utils.get_logger()
 
+C = symbol_shorthand.A
```

Comment:
nit: can we add an inline coment saying `C represents camera variables, and L represents landmark variables`

Reply:
Noting as resolved, thanks.